lib/datadog/tracing/configuration/settings.rb

```diff
@@ -90,13 +90,25 @@ def self.extended(base)
               option :propagation_style do |o|
                 o.type :array
                 o.env Configuration::Ext::Distributed::ENV_PROPAGATION_STYLE
+                o.backup_env Configuration::Ext::Distributed::ENV_OTEL_PROPAGATION_STYLE
                 o.default []
                 o.after_set do |styles|
                   next if styles.empty?
 
                   # Make values case-insensitive
                   styles.map!(&:downcase)
 
+                  if effective_env(:propagation_style) == Configuration::Ext::Distributed::ENV_OTEL_PROPAGATION_STYLE
+                    styles.collect! do |s|
+                      s = 'b3' if s == 'b3single'
```

Comment:
`b3single` is not a valid APM Ruby nor OpenTelemetry propagator. Can't we just let it fall through as an invalid case, just like any order unsupported string value? I don't see it being different from `banana`.

Reply:
Ahhh good point. b3single/b3 single header is a Datadog naming convention and it is deprecated (in some languages). It seems like some ddtrace tracers (ex: java) map propagation_style=b3 to b3 multi headers and propagation_style=b3single  to b3 headers. This was the source of confusion.